chaoslib/openebs/cstor_replica_network_delay.yaml

```diff
@@ -0,0 +1,191 @@
+#- name: Setup pumba chaos infrastructure
+#  shell: >
+#    kubectl apply -f /chaoslib/pumba/pumba_kube.yaml
+#    -n {{ app_ns }}
+#  args:
+#    executable: /bin/bash
+#  register: result
+
+#- name: Confirm that the pumba ds is running on all nodes
+#  shell: >
+#    kubectl get pod -l app=pumba
+#    --no-headers -o custom-columns=:status.phase
+#    -n {{ app_ns }} | sort | uniq
+#  args:
+#    executable: /bin/bash
+#  register: result
+#  until: "result.stdout == 'Running'"
+#  delay: 20
+#  retries: 15
+
+- name: Derive PV from application PVC
+  shell: >
+    kubectl get pvc {{ app_pvc }}
+    -o custom-columns=:spec.volumeName -n {{ app_ns }}
+    --no-headers
+  args:
+    executable: /bin/bash
+  register: pv
+
+- name: Get istgt target pod details
+  #shell: kubectl get pods  -l openebs.io/persistent-volume-claim="{{ APP_PVC }}" --all-namespaces -o custom-columns=:metadata.name
+  shell: kubectl get pods  -l openebs.io/persistent-volume-claim="percona-mysql-claim" --all-namespaces --no-headers -o custom-columns=:metadata.name
+  register: istgt_replica
+
+#kubectl get pods -n app-percona-ns  -l 'name=percona'  --no-headers -o custom-columns=:metadata.name
+- name: Get Application  pod details
+  #shell: kubectl get pods  -l openebs.io/persistent-volume-claim="{{ APP_PVC }}" --all-namespaces -o custom-columns=:metadata.name
+  shell: kubectl get pods -n {{ app_ns }}  -l {{ APP_LABEL }}  --no-headers -o custom-columns=:metadata.name
+  register: app_pod
+
+- name: Get list of cstor replicas pod belonging to the PV
+  shell: >
+    kubectl get pods -l app=cstor-pool
+    -n {{ operator_ns }} --no-headers | awk {'print $1'}
+  args:
+    executable: /bin/bash
+  register: cstor_replicas_pod
+
+#Select any cstor replica pod to inject packet loss
+- set_fact:
+    targeted_replica_pod: "{{ cstor_replicas_pod.stdout_lines | list | first }}"
+
+- name: Install tc command on targeted replica pod
+  shell: kubectl exec -it {{ targeted_replica_pod }} -n  openebs --container cstor-pool -- bash -c "apt-get update && apt-get -y install iproute2"
+  register: tc_apt_output
+
+- name: Inject packet loss on targeted replica
+  shell: kubectl exec -it {{ targeted_replica_pod }} -n  openebs --container cstor-pool -- bash -c "tc qdisc add dev eth0 root netem loss 100.00"
```

Comment:
@ksatchit is this approach of running `tc` directly is fine without going through pumba approach?